### PR TITLE
Feat: try @tiptap/react html based contenteditable with markdown support

### PR DESCRIPTION
**Pull request by rajatkantinandi**
- [x] render editor, handle change, render saved html
- [x] Styling
- [x] Editor toolbar
- [x] Sanitize html using Dompurify

## Screenshot
Screenshot 2023-06-26 at 12 22 19 PM

### CODE REVIEW

File: src/components/FormInputs/ContentEditable.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      Image.configure({
+        allowBase64: false,
+        inline: true,
+      }),
+      Underline,
+      Iframe,
+    ],
+    content: value,
+    enableCoreExtensions: true,
+    autofocus: autofocus,
```
Comment (govind007) [inline src/components/FormInputs/ContentEditable.tsx]: use shorthand

File: src/components/FormInputs/EditorToolbar.tsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+  commandName?: keyof ChainedCommands;
+  children: React.ReactNode;
+  actionName?: string;
+  title?: string;
+  actionParams?: Record<string, any>;
+  onPress?: () => void;
+};
+
+export default function EditorToolbar({ editor, isFocussed }: Props) {
+  const handleLink = useCallback(() => {
```
Comment (govind007) [inline src/components/FormInputs/EditorToolbar.tsx:26]: why we need to wrap it in `useCallback`

File: src/helpers/TipTapIframe.ts [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+       * Add an iframe
+       */
+      setIframe: (options: { src: string }) => ReturnType,
+    }
+  }
+}
+
+export default Node.create<IframeOptions>({
+  name: 'iframe',
+
```
Comment (govind007) [inline src/helpers/TipTapIframe.ts]: remove extra space

File: src/components/FormInputs/EditorToolbar.tsx (hunk 2 above)
Comment (rajatkantinandi) [reply src/components/FormInputs/EditorToolbar.tsx:26]: for optimization

File: src/helpers/TipTapIframe.ts (hunk 3 above)
Comment (rajatkantinandi) [reply src/helpers/TipTapIframe.ts]: removed

**Pull request merged**